### DISCUSSION TITLE
Custom layer manager component icon

### DISCUSSION THREAD

**Discussion by adarshsingh197** [GrapesJS/grapesjs#5652] [Ideas]
i want to change ui of layer manager and add an icon of particular component before there name just like how it is implemented in- https://studio.grapesjs.com/, i have already gone through document i am not able to understand, can someone please help throught this and i am using vue 3.0 with typescript.please help me in this.

**Comment by artf** [comment]
The default editor UI should already support the [`icon` property](https://grapesjs.com/docs/api/component.html#properties) from the Component definition
